src/widgets/SequentialSteps.js

```diff
@@ -144,6 +144,8 @@ SequentialSteps.propTypes = {
   currentStepKey: PropTypes.string.isRequired,
 };
 
+export default SequentialSteps;
```

Comment:
Redundant?

Reply:
Yeah, all components seem to have this set up so I've just kept going with  for consistency..